catalogue/webapp/components/SearchForm/SearchForm.js

```diff
@@ -74,6 +74,26 @@ const SearchForm = ({
 
   function updateUrl(unfilteredSearchResults, form) {
     const workType = searchParams.workType || [];
+    const productionDatesFromInput =
+      form.current && form.current['productionDatesFrom'];
```

Comment:
Where we call this function, we could pass the `event.currentTarget` through instead of the `ref`.